docs/quickstart.rst

```diff
@@ -0,0 +1,135 @@
+Quick Start
+===========
+
+This guide assumes you have a Python project that you wish to use ``towncrier`` on, to generate its news files/changelogs.
+It will cover setting up your project with a basic configuration, which you can then feel free to `customise <customisation/index.html>`_.
+
+Configuration
+-------------
+
+``towncrier`` keeps its config in the `PEP-518 <https://www.python.org/dev/peps/pep-0518/>`_ ``pyproject.toml`` file.
+
+The most basic configuration is annotated below::
+
+    [tool.towncrier]
+        # The name of your Python package
+        package = "myproject"
+        # The path to your Python package.
+        # If your package lives in 'src/myproject/', it must be 'src',
+        # but if you don't keep your code in a 'src' dir, remove the
+        # config option
+        package_dir = "src"
+        # Where you want your news files to come out. This can be .rst
+        # or .md, towncrier's default template works with both.
+        filename = "NEWS.rst"
+
+By default, ``towncrier`` will look for news fragments inside your Python package, in a directory named ``newsfragments``.
+With this example project, it will look in ``src/myproject/newsfragments/`` for them.
+
+Create this folder::
+
+    $ mkdir src/myproject/newsfragments/
+    # This makes sure that Git will never delete the empty folder
+    $ echo "!.gitignore" > src/myproject/newsfragments/.gitignore
+
+The ``.gitignore`` will remain and keep Git from not tracking the directory.
```

Comment:
"The ``.gitignore`` will remain" - when? in which case? "after all fragments have been consumed"?

"and keep Git from not tracking the directory" – I do not understand this. Maybe: "and make Git track this directory"? "and ensure the directory is present after checkout"?